# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/gl/GLContextProviderNull.cpp
# Commit: 204b899f436d
# Full Hash: 204b899f436d99f0eae759af3ccfcbaabfbbb428
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-11 09:34:54
# Regressor Bug: 1632249
# File Overlap Count: 3
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r 3f5baab34b1f -r 204b899f436d gfx/gl/GLContextProviderNull.cpp
--- a/gfx/gl/GLContextProviderNull.cpp	Thu Jun 11 06:36:55 2020 +0000
+++ b/gfx/gl/GLContextProviderNull.cpp	Thu Jun 11 06:37:35 2020 +0000
@@ -22,14 +22,14 @@
 }
 
 already_AddRefed<GLContext> GLContextProviderNull::CreateOffscreen(
-    const gfx::IntSize&, const SurfaceCaps&, CreateContextFlags,
+    const gfx::IntSize&, const GLContextCreateDesc&,
     nsACString* const out_failureId) {
   *out_failureId = NS_LITERAL_CSTRING("FEATURE_FAILURE_NULL");
   return nullptr;
 }
 
 already_AddRefed<GLContext> GLContextProviderNull::CreateHeadless(
-    CreateContextFlags, nsACString* const out_failureId) {
+    const GLContextCreateDesc&, nsACString* const out_failureId) {
   *out_failureId = NS_LITERAL_CSTRING("FEATURE_FAILURE_NULL");
   return nullptr;
 }